src/server.ts

```diff
@@ -27,6 +27,9 @@ const makeCommonEntities = (config: CommonConfig) => {
     ? config.logger
     : new BuiltinLogger(config.logger);
   rootLogger.debug("Running", process.env.TSUP_BUILD || "from sources");
+  process.on("deprecation", (error: Error) =>
```

Comment:
this affects multiple unit tests because of the same process.
should be rather tested in system one.

Reply:
done